starter-code/css/main.css

```diff
@@ -6,3 +6,160 @@ Green: #00B172
 White: #FFF
 
 */
+
+* {
+    margin: 0;
+    padding: 0;
+    font-family: Avenir;
+}
+
+body {
+    width: 100vw;
+    height: 100vh;
+}
+
+header {
+    height: 100px;
+    position: fixed;
+    width: 100%;
+    top: 0;
+    background-color: #fff;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    box-shadow: 0px 7px 17px 1px rgba(0,0,0,0.15);
+}
+
+header > img {
+    width: 200px;
+    height: auto;
+    float: left;
+}
+
+.spotify-logo {
+    margin-left: 40px;
+}
+
+.menu {
+    float: right;
+    margin-right: 40px;
+}
+
+.menu a {
+    text-decoration: none;
+    color: #1A1A1A;
+}
+
+.menu a:hover {
+    color: #00B172;
+    border-bottom: 3px solid #00B172;
+}
+
+.menu li {
+    display: inline-block;
+    padding: 15px;
+    font-size: 20px;
+}
+
+.title-img {
+    background-image: url(../images/landing.jpg);
+    background-repeat: no-repeat;
+    background-size: cover;
+    height: 800px ;
+    padding-top: 100px ; 
+    color: white;
+    margin-bottom: 50px;
+}
+
+.title-img h1 {
+    text-align: center;
+    font-size: 65px; 
+    letter-spacing: 3px;
+    margin-top: 300px;
```

Comment:
This is the property that was messing with your flow and adding a weird white space above when we checked the code together. If you replace it with a padding-top, then you can remove the top: 0 property from the header.